Bottom: fd757ff2ef7cf8485c3e558d91dc74afaaec9617
Top:    7d1529033c1ec49df02e1e5aa4d0b0a536dcdf50
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:26:13 +0000

Fix board outline generation to work when we find nothing

This was hit due to another bug, but since NULL is a valid polygon, we
should handle this case!


---

diff --git a/src/polygon.c b/src/polygon.c
index be4fcf2..f6ae8ab 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2459,6 +2459,12 @@ POLYAREA *board_outline_poly (bool include_holes)
   return clipped;
 #endif
 
+  if (clipped == NULL)
+    {
+      fprintf (stderr, "clipped == NULL in board_outline_poly\n");
+      return clipped;
+    }
+
   /* Now we just need to work out which pieces of polygon are inside
      and outside the board! */
